services/common/src/components/common/CoreTableCommonColumns.tsx

```diff
@@ -8,16 +8,17 @@ import { CaretDownOutlined } from "@ant-design/icons";
 import { generateActionMenuItems } from "./ActionMenu";
 
 export const renderTextColumn = (
-  dataIndex: string,
+  dataIndex: string | string[],
```

Comment:
Could actually change all of the dataIndex params in this file to accept string array. Antd allows access to nested objects with this! Less transforming data, which is nice.